packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -1060,11 +1046,7 @@ export const V1_STORES = [
 export const V3_STORES = V1_STORES;
 
 // Visible for testing
-export const V4_STORES = [
-  ...V3_STORES,
-  DbClientMetadata.store,
-  DbRemoteDocumentChanges.store
-];
+export const V4_STORES = [...V3_STORES, DbClientMetadata.store];
```

Comment:
Hrm...  I'm having uneasy feelings about revising history like this.  I wonder if we should keep some hint about the prior existence of this store to keep our history of schema changes in-tact in this file.  I'm not sure though.  It also seems strange to keep dead code.  Maybe ask @wilhuff what he thinks.